.gitlab-ci.yml

```diff
@@ -1,11 +1,16 @@
-netlify:
-  type: deploy
+image: python:3-slim-stretch
```

Comment:
Are we using shared runners for this instead of our own?

Reply:
I disabled shared runners for this project, so it's using our own.

Reply:
then we don't need `image` and it shouldn't be running any `apt-get` things for every run.